src/core/js/views/blockView.js

```diff
@@ -1,16 +1,22 @@
 define([
     'core/js/views/adaptView'
-    ], function(AdaptView) {
+], function(AdaptView) {
 
     var BlockView = AdaptView.extend({
 
+        attributes: function() {
```

Comment:
You're quite right, better to have in adaptView. For some reason I thought adaptView was inherited more widely than it actually is.